### PR TITLE
Re-enable top level reordering on outline

### PR DESCRIPTION
**Pull request by ibebic**

### CODE REVIEW

**Comment by ibebic** [comment]
This resolves #140.

**Comment by kjuej** [comment]
@ibebic Seems to be working correctly now.

File: client/utils/reorder.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { mapActions } from 'vuex-module';
```
Comment (underscope) [inline client/utils/reorder.js]: Please move it within outline folder and rename it to reorderMixin (unless you had more ideas where you could include it).

File: client/components/course/Outline/Activity.vue [hunk 2]
```diff
@@ -86,7 +88,6 @@ export default {
   },
   methods: {
     ...mapMutations(['focusActivity', 'toggleActivity'], 'course'),
-    ...mapActions({ updatePosition: 'reorder' }, 'activities'),
```
Comment (underscope) [inline client/components/course/Outline/Activity.vue]: Have you accidentally kept reorder function? (Bellow)

Comment (ibebic) [reply client/components/course/Outline/Activity.vue]: My bad 🙈

**Comment by underscope** [comment]
Thanks!

**Pull request merged**